-- ollyVanly_asciirSource: Psilu=_ sipo, A,lal a = (x) return x+1 end
function a.b.c:f2 (x,y) self[x] = y and
assert(a.b.c.f1(4) == 5)
a.b.c:f2('k', 12); assert(a.b.c.k == 12)

print('+ {b={c={}}}

function a.b.c.fon_n (n)
  if n==0 then error(); exit(1);
  else err_ox)[r=6,0
do i{i,ieturn x+1 end        i